npc/re/instances/GhostPalace.txt

```diff
@@ -0,0 +1,975 @@
+	mes "I wanted to get rid of a piece of humanity inside of me through you.";
+	npctalk "Sakray: I wanted to get rid of a piece of humanity inside of me through you.";
+	next;
+	mes "[Sakray]";
+	mes "I am quite satisfied with the result.";
+	next;
+	mes "[Thanatos Magic Trace]";
+	mes "Sakray, offer the blood to me now!";
+	npctalk "Thanatos Magic Trace: Sakray, offer the blood to me now!", instance_npcname("Thanatos Magic Trace#dk3");
+	next;
+	mes "[Sakray]";
+	mes "If you were me, what would you do?";
+	next;
+	mes "[Sakray]";
+	mes "Can you set yourself free from this curse?";
+	next;
+	mes "[" + strcharinfo(0) + "]";
+	mes "...";
+	unittalk getcharid(3), "" + strcharinfo(0) + " : ...";
+	next;
+	mes "[Sakray]";
+	mes "Heh heh... I will keep watching you until the day you grow up enough.";
+	next;
+	mes "[Sakray]";
+	mes "I am willing to hunt you later for Thanatos.";
+	cutin "",255;
+	next;
+	mes "[Sakray]";
+	mes "Farewell then.";
+	close2;
+	donpcevent instance_npcname("#tb") + "::OnStart";
+	end;
+}
+
+1@spa,196,46,1	script	#tb	CLEAR_NPC,{
+	end;
+OnStart:
+	disablenpc instance_npcname("Sakray#dk");
+	disablenpc instance_npcname("Thanatos Magic Trace#dk3");
+	enablenpc instance_npcname("#tb");
+	enablenpc instance_npcname("#sv");
+	enablenpc instance_npcname("#tv");
+	initnpctimer;
+	end;
+OnTimer3000:
+	npctalk "Voice of Princess Tiara : What can I do for you?";
+	end;
+OnTimer8000:
+	npctalk "Voice of Sakray : I will always stand by you. Please rest beside me.", instance_npcname("#sv");
+	end;
+OnTimer13000:
+	npctalk "Thanatos Magic Trace: Sakray, Do not forget the covenant of blood between us.", instance_npcname("#tv");
+	end;
+OnTimer17000:
+	npctalk "Thanatos Magic Trace: Your soul is mine until you find the blood that satisfies me...", instance_npcname("#tv");
+	end;
+OnTimer22000:
+	npctalk "Voice of Sakray : Rest in the shadow with me.", instance_npcname("#sv");
+	end;
+OnTimer25000:
+	npctalk "Voice of Sakray : In the dark forever...", instance_npcname("#sv");
+	end;
+OnTimer28500:
+	disablenpc instance_npcname("#tb");
+	disablenpc instance_npcname("#sv");
+	disablenpc instance_npcname("#tv");
+	mapannounce 'map_spa$, "It is time to leave the Ghost Palace.", bc_map,0xFFFF00,FW_NORMAL,12;
+	enablenpc instance_npcname("Way out of Palace");
+	enablenpc instance_npcname("King#dk11");
+	end;
+}
+
+1@spa,195,41,1	duplicate(Lurid Royal Guard#dk)	Thanatos Magic Trace#dk3	CLEAR_NPC
+1@spa,198,43,1	duplicate(Lurid Royal Guard#dk)	#sv	CLEAR_NPC
+1@spa,196,41,1	duplicate(Lurid Royal Guard#dk)	#tv	CLEAR_NPC
+
+1@spa,204,29,1	script	King#dk11	4_M_RUSKING,{
+	disable_items;
+	mes "[King]";
+	mes "Hey there, are you alive?";
+	mes "I did not expect to meet someone alive";
+	next;
+	mes "[King]";
+	mes "Do you possibly have ^006400Gray Shard^000000?";
+	mes "Can you give it to me?";
+	mes "The old memory of faituful royal guard,";
+	mes "the memory of everyone he loved";
+	mes "becomes broken pieces and are contained inside of it.";
```

Comment:
`The memories of the faithful royal guard`
`and the memories of everyone he loved`
`are contained inside of these shards.`